types/types.d.ts

```diff
@@ -23,6 +23,10 @@ export type ErrorOverlayOptions = {
    * The socket port to use (WDS only).
    */
   sockPort?: number;
+  /**
+   * The socket protocol to use (WDS only).
+   */
+  sockProtocol?: number;
```

Comment:
Hmm I don't think this is right? Can you run the `generate-types` command to regenerate these files?

Reply:
The problem was I didn’t run the command and did it manually. Good catch. Ran the command and it seems to be ok now!